specification/2.0/schema/image.schema.json

```diff
@@ -29,5 +28,8 @@
     "additionalProperties": false,
     "dependencies": {
         "bufferView": [ "mimeType" ]
+    },
+    "not": {
```

Comment:
Just tried with NJsonSchema and gltf-typescript-generator and both seems to ignore the oneOf which is probably the correct behavior. +1 for me.